src/test/java/com/googleinterns/zoomtube/utils/CommentUtilTest.java

```diff
@@ -76,15 +76,15 @@ public void createComment_shouldReturnCommentFromEntity() throws IOException {
   @Test
   public void createEntity_noParent_shouldReturnEntityWithProperties_noParent() throws IOException {
     Key lectureKey = KeyFactory.createKey(LectureUtil.KIND, 12345);
-    Date timestamp = new Date(123);
+    long timestamp = 123;
```

Comment:
Probably should have caught this in a different pr, but should this be a global constant since it is used in multiple tests?

Reply:
Gonna keep as is, because all other tests in this class are using only local variables.

Also not sure what the setup section would look like without em all